code/modules/vending/wardrobes.dm

```diff
@@ -153,6 +153,7 @@
 		/obj/item/clothing/mask/bandana/striped/cargo = 3,
 		/obj/item/clothing/head/soft = 3,
 		/obj/item/radio/headset/headset_cargo = 3,
+		/obj/item/clothing/glasses/hud/gun_permit = 5, //SKYRAT ADDITION
```

Comment:
cargodrobe already has 4 
![image](https://user-images.githubusercontent.com/41448081/174137557-354e3bea-e51a-4f0a-807d-045b0c8ce800.png)

Reply:
I did not see them last time I checked the wardrobe...